Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "meta-socket"
```

Comment:
I usually see the name of the repos with -py suffix to indicate it is a python module, but you typically don't want to install 'meta-memcache-py" as module name, or impots from `meta_memcache_py`, right? But I'm not too familiarized with python packaging....

Reply:
My preference would also be `meta-socket` since this is the rust part. But happy with whatever you go with

Reply:
Also would like to suggest refactoring to call it `meta-memcache-socket` - which unfortunately might mean the repo needs a rename.. and the calling code..... but it would make it explicit what this is for. Again v happy if we don't do that, just had the idea while reviewing